tsconfig.node.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "tsBuildInfoFile": "./node_modules/.tmp/tsconfig.node.tsbuildinfo",
+    "target": "ES2022",
+    "lib": ["ES2023"],
+    "module": "ESNext",
+    "skipLibCheck": true,
+
+    /* Bundler mode */
```

Comment:
The file type here is JSON but these comments are not really valid in this context or type. I would personally suggest removing them.

Reply:
They were part of the scaffolding of `create-vite`. Apparently the tsconfig is actually a JSONC (JSON with Comments) file, so it is actually valid.